saleor/app/tasks.py

```diff
@@ -18,7 +18,13 @@
 
 @celeryconf.app.task
 def install_app_task(job_id, activate=False):
-    app_installation = AppInstallation.objects.get(id=job_id)
+    try:
+        app_installation = AppInstallation.objects.get(id=job_id)
+    except AppInstallation.DoesNotExist:
```

Comment:
Do we know a case when it happens (what causes it)?

Reply:
Our test database in Sentry gets cleared every now and then and it causes Celery to crash. Shouldn't normally happen on production environments but wanted to clear this so Sentry doesn't show it as unhandled 😁